src/components/AnnouncementsModal.js

```diff
@@ -0,0 +1,305 @@
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import { useCookies } from 'react-cookie';
+import { useTranslation } from 'react-i18next';
+import { Box } from './publiq-ui/Box';
+import { Button } from './publiq-ui/Button';
+import { Icon, Icons } from './publiq-ui/Icon';
+import { Image } from './publiq-ui/Image';
+import { Inline } from './publiq-ui/Inline';
+import { Link } from './publiq-ui/Link';
+import { List } from './publiq-ui/List';
+import { ListItem } from './publiq-ui/ListItem';
+import { ModalContent } from './publiq-ui/ModalContent';
+import { Stack } from './publiq-ui/Stack';
+import { getValueFromTheme } from './publiq-ui/theme';
+import { Title } from './publiq-ui/Title';
+
+const getValueForAnnouncementItem = getValueFromTheme('announcementItem');
+const getValueForAnnouncementList = getValueFromTheme('announcementList');
+const getValueForAnnouncementContent = getValueFromTheme('announcementContent');
+
+const AnnouncementItem = ({
+  id,
+  title,
+  selected,
+  seen,
+  setSelectedAnnouncementId,
```

Comment:
I would add an onClick here and link the handlers on the surrounding component instead of passing down the setter